packages/core/integration-tests/test/integration/env-computed/index.js

```diff
@@ -1,3 +1,2 @@
 let name = "ABC";
-process.env[name] = "abc";
```

Comment:
Do we still want to allow mutations with computed properties?? Seems weird to allow that but not static properties.

Reply:
It will always be possible to mutate them anyway by assigning `process.env` to a variable and then changing that.

Reply:
Sure, but the process.env object won't have any properties besides the ones added at runtime, so it's not great to rely on that anyway. IMO better to be consistent when it's not assigned to a variable unless there's a really good reason to allow this.